Roman_Gvozdov/js/scripts.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Created by John Doe on 17.10.2017.
+ */
+document.onreadystatechange = function () {
+  if (document.readyState === 'interactive') {
+    initCalculator();
+  }
+};
+
+function initCalculator() {
+  var
+    calculator = document.getElementById('calculator'),
+    first = document.getElementById('first-number'),
+    second = document.getElementById('second-number'),
+    action = document.getElementById('action');
+
+  calculator.onsubmit = function(event) {
+	 
+    var
+      firstNumber = parseFloat(first.value),
+      secondNumber = parseFloat(second.value),
+      actionType = action.value.trim(),
+      result = calculate(firstNumber, secondNumber, actionType);
+    event.preventDefault();
+	while(typeof(result)!='Nan' || typeof(result)!='undefined'){
+		resultRender(firstNumber, secondNumber, actionType, result);
+		this.reset();
+		}
+  }
+}
+
+function calculate(firstNumber, secondNumber, action) {
+  var result;
+
+  /*if (action === '+') {
+    result = firstNumber + secondNumber;
+  } else if (action === '-') {
+    result = firstNumber - secondNumber;
+  } else if (action === '*') {
+    result = firstNumber * secondNumber;
+  } else if (action === '/') {
+    result = firstNumber / secondNumber;
+  }*/
+  switch(action){
+	  case '+': result = firstNumber + secondNumber;
+	  [break]
```

Comment:
This code throw `"Uncaught SyntaxError: Unexpected token break"` and Your program isn't work.
Please, fix it.